qt/scientific_interfaces/ISISReflectometry/Reduction/LookupTable.h

```diff
@@ -28,9 +28,10 @@ class MANTIDQT_ISISREFLECTOMETRY_DLL LookupTable {
   LookupTable(std::vector<LookupRow> rowsIn);
   LookupTable(std::initializer_list<LookupRow> rowsIn);
 
-  std::vector<LookupRow> const &rows() const;
   boost::optional<LookupRow> findLookupRow(Row const &row, double tolerance) const;
   boost::optional<LookupRow> findWildcardLookupRow() const;
+  ptrdiff_t getIndex(LookupRow const &) const;
```

Comment:
Does this need to be `public`? I'd envisaged the caller would just pass a `Row` rather than having to find the `LookupRow` first so the call to find the index could be internal?

Reply:
We came to the conclusion that giving it a `Row` was breaking and/or as it had to both find the row and then get the index of it, which is an implementation detail that the LookupTable doesn't need to care about, and can just be handled by the calling class

Reply:
Since we already have a `findLookupRow` that returns a `boost::optional`

Reply:
:+1: